package.json

```diff
@@ -39,7 +39,7 @@
     "localforage-observable": "2.0.1",
     "mark.js": "8.11.1",
     "marked": "github:mattermost/marked#87769262aa02e1784570f61f4f962050e07cc335",
-    "mattermost-redux": "github:mattermost/mattermost-redux#26d73f80d7bd4c73e65bc3168058a50cbcae251b",
+    "mattermost-redux": "github:mattermost/mattermost-redux#191efaea2a359933c5bf0279de53448bfe9bd1e3",
```

Comment:
The package-lock.json also needs to be updated to the corresponding redux hash for it to load properly.